knative-operator/deploy/resources/kn-downloads.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-downloads
+  template:
+    metadata:
+      name: kn-downloads
+      labels:
+        app: serverless-operator
+        component: kn-downloads
+    spec:
+      nodeSelector:
+        kubernetes.io/os: "linux"
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: "NoSchedule"
+      - key: "node.kubernetes.io/unreachable"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      - key: "node.kubernetes.io/not-ready"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      containers:
+      - name: kn-download-server
+        terminationMessagePolicy: FallbackToLogsOnError
```

Comment:
This is the default value, let's not specify it.

Reply:
please correct me if I misunderstood, but I see default value for this field as `File`
https://v1-16.docs.kubernetes.io/docs/tasks/debug-application-cluster/determine-reason-pod-failure/